flake.nix

```diff
@@ -18,7 +18,9 @@
     # Used to provide the cardano-node and cardano-cli executables.
     cardano-node = {
       url = "github:input-output-hk/cardano-node";
-      flake = false;
+    };
+    mithril = {
+      url = "github:input-output-hk/mithril";
```

Comment:
FMI: is that to speed up the node setup?

Reply:
That's right. Very fast node bootstrapping.